internal/app/machined/pkg/runtime/v1alpha1/v1alpha1_sequencer.go

```diff
@@ -218,13 +236,10 @@ func (*Sequencer) Boot(r runtime.Runtime) []runtime.Phase {
 	).Append(
 		"legacyCleanup",
 		CleanupLegacyStaticPodFiles,
-	).Append(
-		"udevSetup",
-		WriteUdevRules,
 	).AppendWhen(
 		r.State().Platform().Mode() != runtime.ModeContainer,
-		"udevd",
-		StartUdevd,
+		"udevSetup",
+		WriteUdevRules,
```

Comment:
do we need to kick `udevd` one more time to activate new rules?

Reply:
no need, udevd seems to be automatically reacting to newly added rules, tested with nvidia